src/Request.class.cpp

```diff
@@ -210,10 +212,12 @@ void Request::checkHeaderFields()
 	if (method == "POST")
 	{
 		doesKeyExist(411, "Content-Length", "Missing header field.");
-		if (_headerFields["Content-Length"].length() > 10)
-			throw ErrorResponse(413, "Try a smaller file");
+		int content_length = atoi(_headerFields["Content-Length"].c_str());
```

Comment:
There is a problem with when the file size is bigger than MAX_INT, but I dont think we will encounter that